_authors/josh.md

```diff
@@ -4,6 +4,6 @@ first_name: Josh
 last_name: Ruihley
 full_name: Josh Ruihley
 redirect_from: "/team/josh/"
-published: false
+published: true
+guest: true
```

Comment:
Yeah, I think the text changes on guest authors to indicate they're not current 18F employees.